assignments/arrays.js

```diff
@@ -64,31 +65,66 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
 console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+let car33 = [];
+for (let i = 0; i < inventory.length; i++) {
+    if (inventory[i].id === 33) {
+        car33.push("Car 33 is a " + inventory[i].car_year + " " + inventory[i].car_make + " " + inventory[i].car_model)
+    }
+}
+console.log(car33);
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = [];
+for (let i = 0; i < inventory.length; i++) {
+    if (i = [inventory.length - 1]) {
+        lastCar.push(inventory[i].car_make)
+        lastCar.push(inventory[i].car_model)
+    }
+}
+
+console.log(lastCar);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
 let carModels = [];
-console.log();
+for (let i = 0; i < inventory.length; i++) {
+    if (i < inventory.length) {
+        carModels.push(inventory[i].car_model);
+        carModels.sort();
```

Comment:
having sort here in your loop means you sort your carModels array every time you add a car to it, this could be pretty unreasonable if you had 200,000 cars. moving it down out of the for loop and before the return or console log will mean it only sorts the one time